lib/osutil/lan.go

```diff
@@ -11,10 +11,23 @@ import (
 )
 
 func GetLans() ([]*net.IPNet, error) {
-	addrs, err := net.InterfaceAddrs()
+	ifs, err := net.Interfaces()
 	if err != nil {
 		return nil, err
 	}
+	addrs := []net.Addr{}
+	for _, currentIf := range ifs {
+		if currentIf.Flags&net.FlagUp != net.FlagUp {
+			continue
+		}
+		currentAddrs, err := currentIf.Addrs()
+		if err != nil {
+			return nil, err
+		}
+		for _, currentAddr := range currentAddrs {
+			addrs = append(addrs, currentAddr)
+		}
```

Comment:
```suggestion
		addrs = append(addrs, currentAddrs...)
```
```suggestion
		for _, currentAddr := range currentAddrs {
			addrs = append(addrs, currentAddr)
		}
```

Reply:
Nice!
I didn’t know about this notation, great!